assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import { Grid, Row, Col } from 'react-bootstrap';
+// Graphql imports
+import { compose, graphql } from 'react-apollo';
+import BrightMirrorFictionQuery from '../graphql/BrightMirrorFictionQuery.graphql';
+// HOC imports
+import withLoadingIndicator from '../components/common/withLoadingIndicator';
+// Components imports
+import FictionHeader from '../components/debate/brightMirror/fictionHeader';
+import FictionBody from '../components/debate/brightMirror/fictionBody';
+// Type imports
+import type { CircleAvatarType } from '../components/debate/brightMirror/circleAvatar';
+import type { FictionHeaderType } from '../components/debate/brightMirror/fictionHeader';
+import type { FictionBodyType } from '../components/debate/brightMirror/fictionBody';
+
+type BrightMirrorFictionType = {
+  // contentLocale: string,
+  data: {
+    fiction: BrightMirrorFictionFragment
+  }
+};
+
+class BrightMirrorFiction extends Component<BrightMirrorFictionType> {
+  render() {
+    // const { contentLocale } = this.props;
+    const { fiction } = this.props.data;
+
+    // Define components props
+    const circleAvatarProps: CircleAvatarType = {
+      username: fiction.creator.username,
+      src:
+        fiction.creator.image && fiction.creator.image.externalUrl
+          ? fiction.creator.image.externalUrl
+          : '/static2/img/icons/avatar.png'
+    };
+
+    const fictionHeaderProps: FictionHeaderType = {
+      authorFullname: fiction.creator.name,
```

Comment:
You need to use creator.displayName and a check for isDeleted, see with @cedrtang he did it correctly in #503

Reply:
You could just pass creator instead of fiction to simplify

Reply:
fixed